index.html

```diff
@@ -80,6 +80,7 @@ <h1>YouTube</h1>
 
 
     <main>
+        <section>
         <div id="video-container">
             <div class="video-template">
                 <img class="video-image" src="https://picsum.photos/400/200" alt="">
```

Comment:
## Missing image alt tag
Please make sure to include alt tag to all images as this improves SEO and its good practice for good code.